tests/ballerina-unit-test/src/test/resources/test-src/object/abstract-object-negative.bal

```diff
@@ -67,3 +67,11 @@ type Person5 abstract object {
 function Person5.getName() returns string {
     return "my name";
 }
+
+// Abstract object with private field
+type Person6 abstract object {
```

Comment:
Shall we add a test case where, an abstract object contains a private method and that object is used as a `object type reference` in another object, and trying to implement/access that private method..

eg:
```ballerina
type Foo abstract object {
    private function getName() returns string;
}

type Bar abstract object {
    Foo*
}

function Bar.getName() returns string {
    returns "bar";
}
```

Reply:
What do you look for in such a test? this will fail at Foo object since we do not allow it.

Reply:
Object type referencing will copy the members from the referenced types. Since getName is a private method and is not allowed, it should not get copied. 

Simply, 
```ballerina
function Bar.getName() returns string {
    returns "bar";
}
```
should give an error saying the getName() does not exists.

Reply:
Will write a test. it won't give the method does not exist because we do not allow private in abstract objects now.

Reply:
Test case added, stopped duplicating private methods while type referencing.